pkg/informer/informer.go

```diff
@@ -134,14 +220,10 @@ func (d *DynamicDiscoverySharedInformerFactory) informerForResourceLockHeld(gvr
 		},
 	})
 
-	if err := inf.Informer().AddIndexers(d.indexers); err != nil {
```

Comment:
What's the implication of setting these in the constructor vs using this AddIndexers call?

Reply:
You can't have an error when doing it in the constructor. You can when using AddIndexers. There is no functional difference.